eisbuk-admin/src/components/slots/SlotForm/SlotForm.tsx

```diff
@@ -51,16 +54,17 @@ const defaultValues = {
 };
 
 const SlotValidation = Yup.object().shape({
-  time: Yup.string().required("Manca l'ora"),
+  time: Yup.string().required(i18n.t("SlotValidations.Time")),
   categories: Yup.array()
     .of(Yup.string().min(1))
-    .required("Scegli almeno una categoria"),
-  type: Yup.string().required("Scegli il tipo di allenamento"),
+    .required(i18n.t("SlotValidations.Category")),
+  type: Yup.string().required(i18n.t("SlotValidations.Training")),
   durations: Yup.array()
     .of(Yup.number().min(1))
-    .required("Devi scegliere almeno una durata"),
+    .required(i18n.t("SlotValidations.Duration")),
 });
-// ***** End Region Form Setup ***** //
+
+/***** End Region Form Setup *****/
```

Comment:
*Not related to the PR*: Ok... the correct way to write these comments is // ***** Region ***** // as ES-Lint would complain otherwise (that's why I've switched from /***** Region *****/). This tells me that ES-Lint isn't working properly on pre-commit and I open up an issue to fix that up.